contracts/pods/PodAdminGateway.sol

```diff
@@ -236,58 +179,61 @@ contract PodAdminGateway is CoreRef, IPodAdminGateway {
     }
 
     /// @notice Admin functionality to turn off pod membership transfer
-    /// @dev Permissioned to GOVERNOR, POD_ADMIN, GUARDIAN and the specific role
+    /// @dev Permissioned to GOVERNOR, POD_ADMIN and the specific pod admin role
     function lockMembershipTransfers(uint256 _podId)
         external
         override
-        hasAnyOfFourRoles(
+        hasAnyOfThreeRoles(
```

Comment:
Locked by default when this factory deploys pods. GUARDIAN should not need unlock/lock powers. GUARDIAN scoped to remove members + veto